### DISCUSSION TITLE
Trouble with Status Check and SysInfo on Unraid

### DISCUSSION THREAD

**Discussion by Tw1sted-Turtle** [Lissy93/dashy#886] [Help and Support] [answered]
Just discovered Dashly and I'm in love. I have everything installed and configured how I'd like but I'm running into a couple issues. (Running in a Docker on Unraid)

1) Status Checks fail 100% of the time. Tried the router, the Unraid host, other dockers, public websites. Doesn't matter. Everything times out. Baffled because I can ping all of the above from the dashly docker console.

2) SysInfo widgets work fine when I access via LAN, but when I access via my subdomain some of the widgets load fine and others give errors:
![firefox_uSI3FSMsVr](https://user-images.githubusercontent.com/90106774/188330874-40bf0460-a494-443a-9369-56de6617c778.png)

**Comment by lissy93** [comment]
Heya @Tw1sted-Turtle 
Sorry you're having issues. Could you share the contents of the browser console also, as there will be an error message there ([here's how](https://github.com/Lissy93/dashy/blob/master/docs/troubleshooting.md#how-to-open-browser-console) if you're unsure), plus the relevant parts of your config.

And just to double check, have you seen the Troubleshooting guide in the [docs](https://github.com/Lissy93/dashy/tree/master/docs)? Specifically [Status Checks Failing](https://github.com/Lissy93/dashy/blob/master/docs/troubleshooting.md#status-checks-failing), and [Widget Errors](https://github.com/Lissy93/dashy/blob/master/docs/troubleshooting.md#widget-errors).

**Comment by Tw1sted-Turtle** [comment, marked as answer]
(Edit) Nevermind I see what I did wrong on the SysInfo widgets. i forgot the `useProxy: true` option on those 4 widgets. 

And with status checks the issue seems to be I wasn't giving it enough time. They fail and timeout out for me immediately after I add them - so I was removing them. But if I give it a couple minutes and refresh the page they work!

My apologies. One of the issues was due to my own forgetfulness and the other due to my impatience.

**Reply by lissy93** [reply]
No worries, glad you got it working!
